ReadMe.md

```diff
@@ -64,17 +87,20 @@ setInterval(function (msg) {
 }, 5000, 'Hello world');
 ```
 
-**timer.clear() / interval.clear()**
-
-The response from `safetimers.setTimeout[At]` and `safetimers.setInterval` are `Timer` and `Interval` objects
-respectively. To cancel a timer or interval, you can call `clear` on it.
+To cancel an `Interval`, use the instance's `clear` method, or pass it as an argument to `clearInterval`:
```

Comment:
I feel like there still needs to be some sort of heading (`**...**` like above) right before this line to help separate it from the `setInterval` method description.